src/bp-media/classes/class-bp-rest-media-endpoint.php

```diff
@@ -1260,6 +1310,8 @@ public function prepare_item_for_response( $media, $request ) {
 			'menu_order'      => $media->menu_order,
 			'date_created'    => $media->date_created,
 			'attachment_data' => $media->attachment_data,
+			'group_name'      => ( isset( $media->group_name ) ? $media->group_name : '' ),
```

Comment:
Do not forget to add fields **visibility** and **group_name** into the method **get_item_schema**.